### ISSUE TITLE
Make Error Mac #2 

### ISSUE THREAD

**Issue by MrDrProfK** [PoC-Consortium/engraver#36]
gcc -Wall -m64 -c -o shabal64-darwin.o shabal64-darwin.s
gcc -D OSX -D AMD64 -Wall -m64 -O3 -mtune=native -D_FILE_OFFSET_BITS=64 -c -o helper64.o helper.c		
gcc -D OSX -D AMD64 -Wall -m64 -O3 -mtune=native -D_FILE_OFFSET_BITS=64 -c -o mshabal_sse4.o mshabal_sse4.c
gcc -D OSX -D AMD64 -Wall -m64 -O3 -mtune=native -D_FILE_OFFSET_BITS=64 -mavx2 -c -o mshabal256_avx2.o mshabal256_avx2.c
gcc -D OSX -D AMD64 -Wall -m64 -O3 -mtune=native -D_FILE_OFFSET_BITS=64 -o plot64 plot.c shabal64-darwin.o helper64.o mshabal_sse4.o mshabal256_avx2.o -lpthread -std=gnu99
plot.c:86:49: warning: format specifies type 'int' but the argument has type
      'off_t' (aka 'long long') [-Wformat]
        printf("Allocating file of size %d.\n", len);
                                        ~~      ^~~
                                        %lld
plot.c:127:16: warning: implicit declaration of function 'aligned_alloc' is
      invalid in C99 [-Wimplicit-function-declaration]
        return aligned_alloc( 4096, nmemb * size );
               ^
plot.c:127:16: warning: incompatible integer to pointer conversion returning
      'int' from a function with result type 'void *' [-Wint-conversion]
        return aligned_alloc( 4096, nmemb * size );
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
plot.c:361:7: warning: format specifies type 'unsigned long' but the argument
      has type 'unsigned long long' [-Wformat]
  ...((uint64_t)(n * threads)) + run, nonces,
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
plot.c:794:66: warning: format specifies type 'long' but the argument has type
      'uint64_t' (aka 'unsigned long long') [-Wformat]
        printf("Resuming at nonce %ld with staggersize %d...\n", startnonce, sta...
                                  ~~~                            ^~~~~~~~~~
                                  %llu
plot.c:799:66: warning: format specifies type 'long' but the argument has type
      'uint64_t' (aka 'unsigned long long') [-Wformat]
        printf("Pre-allocating space for file (%ld bytes)...\n", filesize);
                                               ~~~               ^~~~~~~~
                                               %llu
6 warnings generated.
Undefined symbols for architecture x86_64:
  "_aligned_alloc", referenced from:
      _main in plot-652046.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [plot64] Error 1

MacOS 10.13.4

**Comment by simsalabum** [comment]
Hi, having this issue too on macOS Sierra 10.12.6

**Comment by mmullis** [comment]
there's a pull request that fixes this - #33

**Comment by rico666** [comment]
https://github.com/PoC-Consortium/engraver/releases/tag/v0.9.1
